src/js/main.js

```diff
@@ -83,11 +83,11 @@ Site.HomeAnimation = {
   },
 
   addImage: function(imageIndex) {
-    var _this = this;    
+    var _this = this;
     var srcset = _this.generateSrcset(_this.images[imageIndex]);
-    
+
     // append img tag for image
-    _this.$animation.append('<img src="' + _this.images[imageIndex]['600'] + '" src-set="' + srcset + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
+    _this.$animation.append('<img src="' + _this.images[imageIndex]['1800'] + '" srcset="' + srcset + '" id="animation-image-' + imageIndex + '" class="animation-image" />');
```

Comment:
lol I wrote it all and didnt get the attribute name correct. Tho what is best practice for the default size? 1800 is probably too big as the generic fallback

Reply:
not sure about it 😬